docs/source/index.md

```diff
@@ -1,4 +1,25 @@
-# The Jupyter Notebook
+# Jupyter Notebook Documentation
+
+Welcome to the **Jupyter Notebook** documentation site. **Jupyter Notebook**
+is a lightweight notebook authoring application, and is a part of [Project
+Jupyter](https://docs.jupyter.org/en/latest/), a large umbrella project
+centered around the goal of providing tools for interactive computing
+with [computational notebooks](https://docs.jupyter.org/en/latest/#what-is-a-notebook).
+
+A [computational notebook](https://docs.jupyter.org/en/latest/#what-is-a-notebook)
+is a shareable document that combines computer
+code, plain language descriptions, data, rich visualizations like 3D models,
+charts, graphs and figures, and interactive controls. A notebook, along with
+an editor like **Jupyter Notebook**, provides a fast interactive environment for
```

Comment:
Similar comment to https://github.com/jupyterlab/jupyterlab/pull/14568#discussion_r1201092301: 

Would "web-based notebook environment for interactive computing" or " interactive computing environment that enables users to author notebook documents" or " web application for creating and sharing computational documents" work better than work better to describe "Jupyter Notebook" here than "editor"?

Reply:
Ok, sounds good

Reply:
If the goal here is to reduce word count and simplify the description of Notebook, "editor" is a good option. Alternative I can think of is "computational environment" but it does not communicate its relation to notebook as clearly.